jolly-jellyfish/src/page_maker/models.py

```diff
@@ -31,43 +44,48 @@ def get_user_dir_path(instance, filename):
 
 
 class Webpage(models.Model):
```

Comment:
To be honest I still don't see the point of having Template separate from Webpage, but yes, IIRC Django models support inheritance. It might end up making an empty table for the base class though...